hermeto/core/package_managers/pip/main.py

```diff
@@ -6,15 +6,13 @@
 import os
 import os.path
 import re
-import shutil
```

Comment:
In context of previous patch, just for the record this one is fairly good (yet still simple) example of a commit where code movement is correctly displayed by git (if you haven't used `--color-moved` before) and one doesn't need to focus on those lines in the diff at all :).